server/pbenchinacan/container-build.sh

```diff
@@ -24,9 +24,12 @@ HOSTNAME_F=pbenchinacan
 GITTOP=${GITTOP:-$(git rev-parse --show-toplevel)}
 PBINC_DIR=${GITTOP}/server/pbenchinacan
 
+# Image tag determined from jenkins/branch.name
+BRANCH := $(shell cat ${GITTOP}/jenkins/branch.name)
```

Comment:
Do you need the `$(shell`, or would `$(cat` suffice?  Or, better yet, what about `$(<`?

Reply:
Ah, right, not a Makefile ... thanks.